### ISSUE TITLE
Is there any plan about bilinear_upsampling of 5D Tensor？

### ISSUE THREAD

**Issue by bityangke** [Theano/Theano#5954]
For 3D ConvNets，Tensor shape is 5D （batch_size，channel，depth，height，weight）.
If we want to do something such as 3D Segmentation,  bilinear_upsampling of 5D Tensor is needed.

**Comment by nouiz** [comment]
No. But it should not be very hard to implement. Mostly, change the call to
to conv2d... to conv3d. If you want the fastest to implement, you could
implement just the case with 1d kernel. That would need a third call for
the depth dimensions.

If you do it, it would be great to contribute that back to Theano. You can
modify the current function I think. There is much code sharing between
both 2d and 3d version.

**Comment by bityangke** [comment]
@nouiz Thanks!
I will try it.
